packages/react-app/src/pages/api/customers/[id].ts

```diff
@@ -4,6 +4,12 @@ import * as service from '../../../services/customer.service';
 import { internalServerError, notFound } from '../../../errors';
 import { CustomerSchema } from '../../../models/Customer';
 import middlewares from '../../../middlewares';
+import { RoleRestrictions } from '@app/types/Role';
+
+const restrictions: RoleRestrictions = {
+	PUT: ['admin', 'edit-customer'],
```

Comment:
I'm waffling on restricting `GET` to `admin` as well

Reply:
You currently need it for the dropdown in the DAO Selector.